ethereum/contracts/zksync/Storage.sol

```diff
@@ -140,7 +140,7 @@ struct AppStorage {
     /// yet.
     uint256 l2SystemContractsUpgradeBatchNumber;
     /// @dev Address which will exercise non-critical changes to the Diamond Proxy (changing validator set & unfreezing)
-    address admin;
+    address __DEPRECATED_admin;
```

Comment:
If you want to deprecate `admin`, you should also deprecate `pendingAdmin`. Also you can just remove both `admin`/`pendingAdmin` because contracts was never deployed

Reply:
@vladbochok but it will be deployed as this PR will not be part of the boojum upgrade